R/parse.R

```diff
@@ -24,22 +26,31 @@ parse_package <- function(base_path, load_code, registry, global_options = list(
 #'   This can be overridden locally by the tags '@rd` and `@NoRd` inside a
 #'   Roxygen comment.
 #' @param file_encoding The file encoding. Default: `"UTF-8"`.
-#' @param text Instead of specifying `file`, users can also specify a character
-#' vector `text`, containing the R code.
 #' @return A list of roxygen2 blocks.
 #' @export
-parse_code <- function(file, env, registry = default_tags(), wrap = FALSE,
-                       markdown = markdown_global_default,
-                       file_encoding = "UTF-8", text) {
-  if (missing(file) && ! missing(text)) {
+#' @keywords internal
+parse_code <- function(
+  file, text = NULL,
```

Comment:
Oh except the rule for function blocks is that you indent with the `(` in function. I need to document that in style.